framework/tests/viewpointtest/VarargsConstructor.java

```diff
@@ -0,0 +1,46 @@
+// Test case for EISOP issue #777:
+// https://github.com/eisop/checker-framework/issues/777
+import viewpointtest.quals.*;
+
+public class VarargsConstructor {
+
+    VarargsConstructor(String str, Object... args) {}
+
+    @SuppressWarnings({"inconsistent.constructor.type", "super.invocation.invalid"})
```

Comment:
@wmdietl I feel like this is also happening when annotate the constructor in here https://github.com/eisop/checker-framework/blob/481040f5af69012d3b410e1844f31d84fff7b7c4/framework/tests/viewpointtest/PolyConstructor.java#L6. 

can you share some insight on these two error messages?

Reply:
Do the full error messages not explain the errors well enough?
1.) `Constructor type (%s) is a subtype of the top type (%s), therefore it cannot be statically verified.` We cannot statically ensure that the constructor creates an object of a non-top type. Therefore, any constructor that is annotated with something non-top produces this.
2.) `Constructor of type %s cannot call %s of type %s.` The constructor from the superclass has the default annotation. The super constructor is implicitly called and this constructor has a different annotation - therefore, we get a warning about the mismatch.

Please open PRs to improve these error messages if something is unclear.

I notice that for these two errors there is no explanation in `docs/manual`.
We could improve our checks to:
1.) produce a list of all unique error keys from the `messages.properties` files.
2.) ensure each key appears somewhere in `docs/manual`. Ideally, outside of just a code example (e.g. one of the two error keys appears once as `@SuppressWarnings("super.invocation.invalid")`. Is that good enough as documentation?)
3.) ensure each key appears somewhere in the checker source code, to avoid that we have dead error messages in `messages.properties` files.
4.) ensure each key appears in at least one test case as expected error.

I'm not sure how well-covered the manual is currently for the error keys, but this looks like a nice goal to reach.
If this sounds useful, please convert this comment into an issue.